apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -193,12 +194,21 @@ var TopInstructions = React.createClass({
   adjustMaxNeededHeight() {
     const minHeight = this.getMinHeight();
 
-    const instructionsContent = this.refs.instructions;
-    const maxNeededHeight = $(ReactDOM.findDOMNode(instructionsContent)).outerHeight(true) +
-      RESIZER_HEIGHT;
+    if (this.props.hasContainedLevels) {
+      const maxNeededContainedLevelHeight =
+        $(ReactDOM.findDOMNode(this)).find('#containedLevelContainer').outerHeight(true) +
```

Comment:
Good suggestion - now that I see how that works, it actually make the code smaller for this function.